### DISCUSSION TITLE
A new life of handcuffs

### DISCUSSION THREAD

**Discussion by MrPofigist** [FakeFishGames/Barotrauma#6182] [Ideas and suggestions]
Many will agree that handcuffs are of no use when going through the campaign alone or with friends. After the latest updates, we have a lot of missions related to hostile NPCs.

How about introducing the ability to increase the mission reward for having a submarine crew arrest an enemy submarine captain/bandit leader at an outpost by taking him alive? For example, if you just clear the outpost/neutralize the submarine, the reward will be, say, 2800, and if you try and capture the leader, the amount will increase to 3500.

From a technical point of view, it is quite possible to do this, because after the update, the game is already able to: track the location of key NPCs (hostages on board), as well as switch the state of NPCs and radically change their behaviour depending on the state of their inventory (transportation of prisoners).

Positive:

- handcuffs will gain their significance in the single-player campaign and in the game with friends

- missions to clean up captured outposts will become even more interesting, as well as gain a variety of passage

Negative:

-further development will be needed

**Comment by James-E-A** [comment]
Partially implemented in the Ancients update with Security's "By the Book" skill

Possible improvements: scale reward with hostile rank (e.g., subduing Pirate Captain gives more marks than the staff)

**Reply by StruggleDuck** [reply]
I agree, price should scale. Also seeing as how some pirates are not even properly armed, where as some, including the captain, actually will shoot back at you.
